modules/exoplanets/tasks/exoplanetsdatapreparationtask.h

```diff
@@ -25,6 +25,7 @@
 #ifndef __OPENSPACE_MODULE_EXOPLANETS___EXOPLANETSDATAPREPARATIONTASK___H__
 #define __OPENSPACE_MODULE_EXOPLANETS___EXOPLANETSDATAPREPARATIONTASK___H__
 
+#include <modules/exoplanets/exoplanetshelper.h>
 #include <openspace/util/task.h>
```

Comment:
```suggestion
#include <openspace/util/task.h>

#include <modules/exoplanets/exoplanetshelper.h>
```

since we derive from `Task`